spec.html

```diff
@@ -28087,278 +28085,158 @@ <h1>Function Properties of the Math Object</h1>
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.cbrt">
         <h1>Math.cbrt ( _x_ )</h1>
-        <p>Returns an implementation-approximated value representing the result of the cube root of _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-        </ul>
+        <p>Returns the cube root of _x_.</p>
+        <p>When the `Math.cbrt` method is called with argument _x_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *+0*, _n_ is *-0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. Return an implementation-approximated value representing the result of the cube root of _n_.
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.ceil">
         <h1>Math.ceil ( _x_ )</h1>
-        <p>Returns the smallest (closest to *-&infin;*) Number value that is not less than _x_ and is an integer. If _x_ is already an integer, the result is _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-          <li>
-            If _x_ is less than 0 but greater than -1, the result is *-0*.
-          </li>
-        </ul>
-        <p>The value of `Math.ceil(x)` is the same as the value of `-Math.floor(-x)`.</p>
+        <p>Returns the smallest (closest to *-&infin;*) integral Number value that is not less than _x_. If _x_ is already an integral Number, the result is _x_.</p>
+        <p>When the `Math.ceil` method is called with argument _x_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is *NaN*, _n_ is *+0*, _n_ is *-0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+          1. If _n_ &lt; 0 and _n_ &gt; -1, return *-0*.
+          1. If _n_ is an integral Number, return _n_.
+          1. Return the smallest (closest to *-&infin;*) integral Number value that is not less than _n_.
+        </emu-alg>
+        <emu-note>
+          <p>The value of `Math.ceil(x)` is the same as the value of `-Math.floor(-x)`.</p>
```

Comment:
i wonder if it'd be simpler to just have this algorithm return `- %Math.floor(-x)%` rather than repeating the steps and having the note.